### PR TITLE
Enable Utility feature

### PR DESCRIPTION
**Pull request by npt-cparker**

### CODE REVIEW

File: util/config.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+config1:abc
+
+config2:123
```
Comment (chriseboy) [inline util/config.txt]: this config value should be 567. old value was 123

File: util/feature.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+feature 1 implementation
+
+feature 2 implementation
```
Comment (chriseboy) [inline util/feature.txt:3]: let's add a bit more to the feature 2 implementation. I think it should handle multiple widgets

**Review by chriseboy — CHANGES_REQUESTED** [review]
Please fix config value and improve feature 2

File: util/feature.txt (hunk 2 above)
Comment (npt-cparker) [reply util/feature.txt:3]: @chriseboy see the latest commit for implementation

**Review by chriseboy — APPROVED** [review]

**Pull request merged**